_includes/selector.html

```diff
@@ -688,44 +646,38 @@
             },
             disableUnsupportedCuda(cuda_version) {
                 var isDisabled = false;
-                if (this.active_img_options.includes("development") && cuda_version !== this.getNewestCudaVer()) {
-                    isDisabled = true;
-                }
                 if (this.active_additional_packages.includes("TensorFlow") && cuda_version !== "11.2") isDisabled = true;
-                if (this.active_os_ver === 'Ubuntu 22.04' && cuda_version !== '11.8' && this.active_method === 'Docker') isDisabled = true;
-                if (this.active_method !== 'pip' && cuda_version === '12.0') isDisabled = true;
+                if (this.active_arch === "arm" && cuda_version === "12.0") isDisabled = true;
 
                 return isDisabled;
             },
             disableUnsupportedPython(python_version) {
                 var isDisabled = false;
                 return isDisabled;
             },
-            disableUnsupportedImgOS(os) {
+            disableUnsupportedImgType(type) {
                 var isDisabled = false;
-                if (this.active_arch === "arm" && os === "CentOS 7") isDisabled = true;
-                if (this.active_cuda_ver !== '11.8' && os === 'Ubuntu 22.04' && this.active_method === 'Docker') isDisabled = true;
-                if (this.active_arch === "arm" && os === 'Ubuntu 20.04' && this.active_method === 'Docker') isDisabled = true;
+                if (this.active_arch === "arm" && type !== "core") isDisabled = true;
                 return isDisabled;
             },
-            disableUnsupportedImgType(type) {
+            disableUnsupportedImgLoc(loc) {
                 var isDisabled = false;
-                if (this.active_arch === "arm" && type !== "core") isDisabled = true;
                 return isDisabled;
             },
-            disableUnsupportedImgOption(option) {
+            disableUnsupportedMethod(method) {
                 var isDisabled = false;
-                var isNGC = this.active_img_loc === "NGC"
-                if (isNGC && option === "development") isDisabled = true;
+                if (this.active_release === "Nightly" && method === "pip") isDisabled = true;
                 return isDisabled;
             },
-            disableUnsupportedImgLoc(loc) {
+            disableUnsupportedArch(arch) {
                 var isDisabled = false;
+                if (this.active_method === "Conda" && this.active_cuda_ver === "12.0" && arch === "arm") isDisabled = true;
+                if (this.active_method === "Docker" && this.active_cuda_ver === "12.0" && arch === "arm") isDisabled = true;
```

Comment:
Condense this logic a bit. Conda/Docker can be handled together. Either suggestion below is okay.

```suggestion
                if (["Conda", "Docker"].includes(this.active_method) && this.active_cuda_ver.startsWith("12") && arch === "arm") isDisabled = true;
```

```suggestion
                if ((this.active_method === "Conda" || this.active_method === "Docker") && this.active_cuda_ver.startsWith("12") && arch === "arm") isDisabled = true;
```